### ISSUE TITLE
Can't change segmented list item (option) on runtime

### ISSUE THREAD

**Issue by darkwood-studios** [TheJuki/KFormMaster#60]
Hi there,

I do have a simple segmented form element which looks like this
```
var squareMeterList = listOf(
                                   SquareMeterListItem(id = 1, name = "30 m²", usage = "1250"),
                                   SquareMeterListItem(id = 2, name = "50 m²", usage = "2500"),
                                   SquareMeterListItem(id = 3, name = "100 m²", usage = "5000"),
                                   SquareMeterListItem(id = 4, name = "150 m²", usage = "9000")
                           )
segmented<SquareMeterListItem>(FormField.USAGE_BUTTONLIST) {
                options = squareMeterList
                value = squareMeterList[1]
}
```

Now I want to replace the segmented list with a new one at runtime. This is the code:

```
var squareMeterListNew = listOf(
                                   SquareMeterListItem(id = 1, name = "30 m²", usage = "6000"),
                                   SquareMeterListItem(id = 2, name = "50 m²", usage = "10000"),
                                   SquareMeterListItem(id = 3, name = "100 m²", usage = "20000"),
                                   SquareMeterListItem(id = 4, name = "150 m²", usage = "36000")
                           )
formBuilder.getFormElement<FormSegmentedElement<SquareMeterListItem>>(FormField.USAGE_BUTTONLIST).options = squareMeterListNew
```

But the app crashes with this error:

> java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.Object android.content.Context.getSystemService(java.lang.String)' on a null object reference
>         at android.view.LayoutInflater.from(LayoutInflater.java:229)
>         at com.thejuki.kformmaster.model.FormSegmentedElement.reInitGroup(FormSegmentedElement.kt:174)
>         at com.thejuki.kformmaster.model.FormSegmentedElement.reInitGroup$default(FormSegmentedElement.kt:167)
>         at com.thejuki.kformmaster.model.FormSegmentedElement.setOptions(FormSegmentedElement.kt:61)

I would expect that the segmented button element will be rebuilded with the new options.

**Comment by TheJuki** [comment]
I just released [5.0.4](https://github.com/TheJuki/KFormMaster/releases/latest). Please update your dependency and let me know if updating the options works for you now.

**Comment by darkwood-studios** [comment]
@TheJuki 
Thanks for the fast fix - it works now. But there seems to be a problem with setting the new value.

After setting the new option list, I want to pre-select the first one. 

`formBuilder.getFormElement<FormSegmentedElement<SquareMeterListItem>>(FormField.USAGE_BUTTONLIST).setValue(squareMeterListNew[1])`

There is no error, but nothing happens. Do you have any idea for that?

**Comment by TheJuki** [comment]
I just released [5.0.5](https://github.com/TheJuki/KFormMaster/releases/latest). Please update your dependency and let me know if it works now.

**Comment by darkwood-studios** [comment]
work great now. thx.
